airbyte-integrations/connectors/source-mailchimp/sample_files/test_config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
as we discussed today these credentials should be removed from the pull request. if you put them in `airbyte-integrations/connectors/source-mailchimp/secrets/config.json` they will be gitignored. by convention this is where we put credentials when we are testing locally. this is also where the CI assumes credentials are store.

You can see how we take secrets out of github secrets in our CI and write them to the `secrets` directory of each integration in this [script](https://github.com/airbytehq/airbyte/blob/master/tools/bin/ci_credentials.sh). You do not need to worry about updating this script, I just want to give you context into how this all works.